interface/patient_tracker/patient_tracker.php

```diff
@@ -49,7 +49,18 @@
                 (?,?,?,'NOTES','CALLED',?)";
     sqlQuery($sqlINSERT, array($_POST['pc_eid'], $_POST['pc_pid'], $_POST['campaign_uid'], $_POST['txtCALLback']));
 }
-$from_date = !is_null($_REQUEST['form_from_date']) ? DateToYYYYMMDD($_REQUEST['form_from_date']) : date('Y-m-d');
+// dh changing this next line to default to the monday of the current week
+// $from_date = !is_null($_REQUEST['form_from_date']) ? DateToYYYYMMDD($_REQUEST['form_from_date']) : date('Y-m-d');
+
+
+if (!is_null($_REQUEST['form_from_date'])){
+    $from_date = DateToYYYYMMDD($_REQUEST['form_from_date']);
+    
+} elseif (date('w') ==1) {
+    $from_date = date('Y-m-d', strtotime('previous monday'));
+} else {
+    $from_date = date('Y-m-d', strtotime('previous monday'));
+}
```

Comment:
I think this option would be a useful feature. Check out Administration->Globals->Flow Board: Ending Date setting and rec seeing how that is used.
You could then set a Administration->Globals->Flow Board: Starting Date that is set by Default to 'Same Day'. And then your option would be 'Beginning of Work Week', which could be picked in globals. Feel free to take a stab at this and if you need guidance either on where to start or if issues arise, just give me a holler here.

Reply:
I am starting to work on this...